transmart-core-api/src/main/groovy/org/transmartproject/core/userquery/UserQuerySetDiff.groovy

```diff
@@ -0,0 +1,22 @@
+package org.transmartproject.core.userquery
+
+/**
+ *  Stores query result change entries
+ */
+interface UserQuerySetDiff {
+    /**
+     * Internal system identifier of the querySetInstance.
+     */
+    Long getId()
+
+    /**
+     * The id of the object that was updated in the query set e.g. patient.id
+     */
+    Long getObjectId()
+
+    /**
+    *  Flag determining whether the object was added or removed
+    */
+    String getChangeFlag()
```

Comment:
The enum can be used here, right?

Reply:
ChangeFlag enum is a part of core-db, so there is no access from core-api, but I can move the enum class to a different module, if it is needed.

Reply:
Yes, please move it to core-api.